benchmark/single-source/NSDictionaryCastToSwift.swift

```diff
@@ -20,6 +20,7 @@ import TestsUtils
 
 @inline(never)
 public func run_NSDictionaryCastToSwift(_ N: Int) {
+#if _runtime(_ObjC)
```

Comment:
@airspeedswift As described in the comment, ```utils/main.swift``` is generated from templates. My goal is to selectively skip compiling and running each test solely by itself while keeping templates and the generation mechanism as is. I think this is important to avoid any impact to the current benchmark framework.